### DISCUSSION TITLE
Czech translation

### DISCUSSION THREAD

**Discussion by motirek** [GradienceTeam/Gradience#105] [General]
When translating I noticed, that there are some words that have the same meaning in Czech language, e.g. backdrop color and background color. 

Therefore I did not change them and only left suggestions to them. 

If you'd like to make some changes to the translation, just comment here and I will try to get to it ASAP.

**Comment by 0xMRTT** [comment]
Hi @motirek 

Thanks for taking part of Adwaita Manager

You can update translations on [Transifex](https://www.transifex.com/adwcustomizerteam/adwcustomizer/). Your contributions will be very appreciated. Thanks
